### PR TITLE
doc(*): initial spec draft

### PR DESCRIPTION
**Pull request by SergejIsbrecht**

### CODE REVIEW

**Comment by fhilgers** [comment]
Three quick questions regarding licensing:

1. Are you fine with CC-BY-4.0 for the adoc files (more or less a requirement by the course. We can diverge but should look that the code is MIT and other files are CC-BY-4.0)
2. Do you want to be addes explicitly to the copyright holders, otherwise we'll just put 'The AMOS projects' as copyright holder.
3. What license are the icons used throughout the documentation?

**Comment by SergejIsbrecht** [comment]
> Are you fine with CC-BY-4.0 for the adoc files (more or less a requirement by the course. We can diverge but should look that most of the code is MIT and other files are mostly CC-BY-4.0)

Yes, no problem. Just edit as you like

> Do you want to be added explicitly to the copyright holders, otherwise we'll just put 'The AMOS projects' as copyright holder.

AMOS project is fine

> What license are the icons used throughout the documentation?

Good question. I will remove them, because they are not used anyhow.

**Comment by SergejIsbrecht** [comment]
Icons are now removed

File: Documentation/asciidoc/feat/build_aosp.adoc [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+* repo (`sudo apt-get install repo`)
+
+NOTE: export `adb` from installed Android Tools (e.g. `<path>/Android/Sdk/platform-tools/adb`) instead of installing via `apt` (Ubuntu).
+
+### Commands
+
+.check whether tracing syscalls is enabled on an Android device
+[source,bash]
+----
+# adb -e shell -- zcat /proc/config.gz| grep CONFIG_BPF_SYSCALL
```
Comment (SergejIsbrecht) [inline Documentation/asciidoc/feat/build_aosp.adoc:19]: Das muss noch geändert werden auf `adb -e shell -- zcat /proc/config.gz| grep CONFIG_FTRACE_SYSCALLS`

Comment (fhilgers) [reply Documentation/asciidoc/feat/build_aosp.adoc:19]: done in #68

**Comment by fhilgers** [comment]
I have created #68:

- [X] Include License information
- [X] fix syntax highlighting
- [X] replace `grep CONFIG_BPF_SYSCALL` with `grep CONFIG_FTRACE_SYSCALLS`
- [X] add initial CI

**Pull request closed without merging**